eng/common/scripts/check-spelling-in-changed-files.ps1

```diff
@@ -0,0 +1,99 @@
+[CmdletBinding()]
+Param (
+    [Parameter()]
+    [string] $TargetBranch,
+
+    [Parameter()]
+    [string] $SourceBranch,
+
+    [Parameter()]
+    [ValidateNotNullOrEmpty()]
+    [string] $CspellConfigPath = "./.vscode/cspell.json"
+)
+
+$ErrorActionPreference = "Continue"
+. $PSScriptRoot/logging.ps1
+
+if ((Get-Command git | Measure-Object).Count -eq 0) { 
```

Comment:
Gotta say I love this attention to detail ;)

Reply:
@weshaggard deserves credit for the idea of checking for these scripts. @benbp for ensuring that we are consistent in checking for `git`